.github/workflows/helm-lint.yaml

```diff
@@ -0,0 +1,33 @@
+name: check-manifest
+on:
+  push:
+    paths:
+      - "cmd/**"
```

Comment:
do we need this?

Reply:
nm i see `cmd/build/helmify/static/`, can we make it `cmd/build/helmify/static/`